tests/custom-elements/reactions/HTMLButtonElement.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<title>Custom Elements: CEReactions on HTMLButtonElement interface</title>
+<meta name="author" title="Zhang xiaoyu" href="xiaoyux.zhang@intel.com">
+<meta name="assert" content=" autofocus, disabled, formAction, formEnctype, formMethod, formNoValidate, formTarget, name, type, value of HTMLButtonElement interface must have CEReactions">
+<meta name="help" content="https://html.spec.whatwg.org/#htmlbuttonelement">
+<script src="/tests/custom-elements/resources/testharness.js"></script>
+<script src="/tests/custom-elements/resources/testharnessreport.js"></script>
+<script src="./custom-elements-helpers.js"></script>
+<script src="./reactions.js"></script>
+<body>
+<script>
+
+//In parent node, sub node's observeAttribute can enqueue by changing attribute value 
+//Re->reflect, At->attribute, P->parent, Co->content
+function testReAtWithPNodeWithCoValues(jsAtName, coAtName, jsAtValue1, coAtValue1, jsAtValue2, coAtValue2, name, elementName, pElementName, interfaceName) {
```

Comment:
How about `testReflectAttrWithContentValuesAndPNode`? And replace the second line of comment with "Test reflect attribute with content values and parent node", not sure if I am right.